smf/switch_zone_setup/switch_zone_setup

```diff
@@ -21,4 +21,7 @@ done
 pfexec routeadm -e ipv6-forwarding -u
 pfexec routeadm -e ipv6-routing -u
 
+# Add tools like swadm & ddmadm to the PATH by default
+echo 'export PATH=$PATH:/opt/oxide/dendrite/bin:/opt/oxide/mg-ddm/bin' >> ~root/.profile
```

Comment:
I suspect we should just ship a `/root/.profile` file in the zone image that meets your expectations, rather than edit it at runtime.

If you're going to edit it like this, we should at least make it idempotent: if this service runs more than once, it will add this line more than once to the file, and `PATH` will get longer and longer.

Reply:
Added it as part of the switch zone image in [58487c3](https://github.com/oxidecomputer/omicron/pull/3028/commits/58487c33f15841e2750eb2ca3130072eb8c392a5).